src/utils/analytics.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+
+const ANALYTICS_ENDPOINT = 'https://analytics.infinit.tech'
```

Comment:
move this to env?

Reply:
let's just hardcode for now, we've never setup the build for `.env` before in the CLI.